store/src/java/com/zimbra/qa/unittest/SolrEventStoreTestBase.java

```diff
@@ -286,7 +288,7 @@ void testContactFrequencyCount(ContactAnalytics.ContactFrequencyTimeRange timeRa
     }
 
     protected void testContactFrequencyGraph(ContactAnalytics.ContactFrequencyGraphTimeRange timeRange, SolrEventCallback eventCallback, String collectionName, SolrEventStore eventStore) throws Exception {
-        List<Timestamp> timestamps = getTimestampForContactFrequencyGraphTest(timeRange);
+        List<Timestamp> timestamps = getTimestampForContactFrequencyGraphTest(timeRange, ZoneId.of("GMT-0800")); //LA time zone
```

Comment:
I assume "GMT+0000" is a valid ZoneId and it doesn't collapse to "GMT"?  Might be worth a test to confirm.

Reply:
Ran test for GMT+0000 timezone. Seems to be working. Thanks for pointing this out. I didn't even know that "GMT+0000" was a valid timezone Id.